install-dependencies.sh

```diff
@@ -67,5 +84,23 @@ elif [ ${UBUNTU_VER} = '18.04' ]; then
     sudo ./b2 -j `nproc` install threading=multi
 fi
 
+if [ ${UBUNTU_VER} = '20.04' ]; then
+  # Install RapidJSON
+  mkdir -p ~/git 
+  cd ~/git && git clone https://github.com/Tencent/rapidjson.git
+  cd rapidjson && mkdir build && cd build
+  cmake .. && make -j `nproc` && make install
```

Comment:
Installing rapidjson and pybind11 fails when running without permissions. Might suggest changing to `sudo make install` for both like osg, fcl, and openrave script files.